### ISSUE TITLE
[Docs]: Search functionality in the doc is not pulling up the most relevant results

### ISSUE THREAD

**Issue by lolzypops** [remix-run/react-router#12645]
### Describe what's incorrect/missing in the documentation

The search functionality in the docs (using cmd + K) no longer pulls up the documentation pages. Some examples: 

`BrowserRouter`

https://github.com/user-attachments/assets/b096b569-45cc-4bee-be77-34a5496c63db

`useLoaderData`

https://github.com/user-attachments/assets/b5ec7371-01f9-401e-8ab9-db4cb5cda230

In both cases, the top results link to patch notes or less relevant documentation - making it difficult for developers to find the info they need on `react-router` functionality. I feel like this has been broken for a month or 2 at this point - maybe there's an existing issue documenting this?

**Comment by corydambach** [comment]
Yeah, I can't find anything good for useLoaderData using the site's search feature.  What's going on with these docs?

**Comment by brookslybrand** [comment]
Sorry this issue has been open for a while. Yeah, our launch of the docs was not great, so we were missing a lot of docs. It wasn't search so much as it was lack of docs.

We've added a lot more docs, so now these searches are useful (except the first one for `useBrowserRouter`, but only because that isn't an API)

We also added search for the [v6 docs for anyone who still needs access to those](https://github.com/remix-run/react-router-website/pull/174)

I'm going to go ahead and close this issue. If you notice other problems related to the website, please open up an issue in the [`react-router-website` repo](https://github.com/remix-run/react-router-website/issues/new)
